src/comparisons/elements/remove/modern.js

```diff
@@ -1 +1,10 @@
 el.remove();
+
+// one element:
```

Comment:
The multiple elements example is exactly how jQuery behaves, so we can omit this one-element example